src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+import {configure} from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+configure({adapter:new Adapter()});
+
```

Comment:
This should preferably be in the root folder

Reply:
Thanks,I have mad the change

Reply:
It is because the `__tests__` are in the src folder. Please take them to the root directory as well

Reply:
@LehruAngela ,I have put them in root but ,the test suite doesnt recognise them.It seems like the create-react-app sets predetermined locations for tests `testMatch: /Users/kimbugwesimon/Desktop/ah-frontend-lannister/src/**/__tests__/**/*.{js,jsx},/Users/kimbugwesimon/Desktop/ah-frontend-lannister/src/**/?(*.)(spec|test).{js,jsx} `
I tried over-riding this but it returns the error below.
<img width="1204" alt="screen shot 2018-10-31 at 12 57 28" src="https://user-images.githubusercontent.com/39520700/47780501-8e353d80-dd0c-11e8-9881-8b6bab016855.png">

Reply:
Oh snap, alright let's leave it like that then.